addon.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="plugin.audio.wimp" name="WiMP" version="1.0.0" provider-name="takoi">
+<addon id="plugin.audio.wimp" name="WiMP" version="1.0.1" provider-name="takoi">
```

Comment:
Dropped it. The reason I added it was that the latest version of the add-on in kodi is 1.0.1. So if you manually import this version, it will automatically update to the older 1.0.1 version that is available from within kodi.